frontend/src/views/Shares/ShareOutboxListItem.js

```diff
@@ -80,51 +79,51 @@ const ShareOutboxListItem = (props) => {
           </Box>
         </Grid>
         <Grid item justifyContent="flex-end" md={3} xs={6} spacing={2}>
-            <Box
-                sx={{
-                  alignItems: 'center',
-                  display: 'flex',
-                  pt: 3,
-                  pb: 0.5
-                }}
-            >
-              <Typography color="textPrimary" variant="body2">
-                {`Currently shared items: ${share.statistics.sharedItems}`}
-              </Typography>
-            </Box>
-            <Box
-                sx={{
-                  alignItems: 'center',
-                  display: 'flex',
-                  py: 0.5,
-                }}
-            >
-              <Typography color="textPrimary" variant="body2">
-                {`Revoked items: ${share.statistics.revokedItems}`}
-              </Typography>
-            </Box>
-            <Box
-                sx={{
-                  alignItems: 'center',
-                  display: 'flex',
-                  py: 0.5,
-                }}
-            >
-              <Typography color="textPrimary" variant="body2">
-                {`Failed items: ${share.statistics.failedItems}`}
-              </Typography>
-            </Box>
-            <Box
-                sx={{
-                  alignItems: 'center',
-                  display: 'flex',
-                  py: 0.5
-                }}
-            >
-              <Typography color="textPrimary" variant="body2">
-                {`Pending items: ${share.statistics.pendingItems}`}
-              </Typography>
-            </Box>
+          <Box
```

Comment:
This code is identical to [frontend/src/views/Shares/ShareInboxListItem.js](https://github.com/awslabs/aws-dataall/pull/394/files#diff-da6d97e0fad1548010c159d1fa44007b2b1316aaa3db2e3d711d0898ede7d402). Let's extract it into dedicated form

Reply:
Refactoring and moving parts around will come in the subsequent PRs, I want this PR to be free from any logic so that it's easy to review and can be merged quickly. But I noted the comment and will apply it when working on the refactoring.